README.md

```diff
@@ -8,6 +8,41 @@ Yet another note taking tool.
 cargo tauri dev
 ```
 
+## Setup environment
+This guide focuses on how to setup your development system based on ubuntu:jammy.
+
+1. Install dependencies
+    ```bash
+    sudo apt-get update
+    sudo apt-get install -y \
+        libwebkit2gtk-4.0-dev \
+        build-essential \
+        curl \
+        wget \
+        file \
+        libssl-dev \
+        libgtk-3-dev \
+        libayatana-appindicator3-dev \
+        librsvg2-dev
+    ```
+2. Install rust
+    ```bash
+    curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+    ```
```

Comment:
This section should be removed, just state that rust is needed. There are multiple ways to install rust depending on your personal preferences and requirements. Do not force a user to a specific way.

```suggestion
```

Reply:
This section is removed with commit c337707716c614b8271430b3999e47c3e33cc656 and only states the need to have it installed beforehand.